datafusion/core/src/dataframe.rs

```diff
@@ -714,6 +737,12 @@ impl DataFrame {
         }
     }
 
+    /// Convert a prepare logical plan into its inner logical plan with all params replaced with their corresponding values
+    pub fn with_param_values(self, param_values: Vec<ScalarValue>) -> Result<Self> {
+        let plan = self.plan.with_param_values(param_values)?;
```

Comment:
I plan to move this off `LogicalPlan` and onto `LogicalPlanBuilder` in a subsequent PR, to be consistent with the rest of the system